### ISSUE TITLE
validating error after zipping:File 'META-INF/container.xml' could not be found'.

### ISSUE THREAD

**Issue by Bahar1978** [lostandfound/epub-zip#2]
Hi, I have a valid epub file. I uziped its package and zip(.pub) it again using epub-zip. but it is not a valid epub. When I validate it it gives me this error:  File 'META-INF/container.xml' could not be found'.
But when I zip it using epubPack it is ok.

**Comment by VlaDavAle** [comment]
Yes, me too have this problem, but only on windows system, if using linux or mac, all ok. Has anyone found a solution to the problem?
